governance/src/components/FAB/StakingFab.tsx

```diff
@@ -139,6 +144,35 @@ const StakingFAB = () => {
     };
   }, [active, loading, loadingText, rbnTokenAccount, veRBNBalance]);
 
+  const lockButtonContent = useMemo(() => {
+    const pendingTx = pendingTransactions.find(
+      (tx) =>
+        [
+          "governanceStake",
```

Comment:
Not sure if it's being used a lot elsewhere, but would it make sense to use an enum to contain these string values?

Reply:
Updated to not use hardcoded strings

<img width="1056" alt="Screenshot 2022-03-03 at 3 25 04 PM" src="https://user-images.githubusercontent.com/11567740/156516678-ce9ca448-e820-4fa1-9a06-31ea8c93d891.png">